html/changelogs/AutoChangelog-pr-6967.yml

```diff
@@ -1,4 +0,0 @@
-author: "nichlas0010"
```

Comment:
This has nothing to do in this PR

Reply:
I have no idea where that came from or how to fix that, i am not seeing it in any of my change history

Reply:
big think

Reply:
maybe you should stop using fucking master for your other PRs

Reply:
and this wouldn't have happened, now would it

Reply:
https://github.com/yogstation13/Yogstation/pull/7015/commits/17d9133de568b126bfb2df7aaf6a479514bfe71e

You explicitly reverted this change.

Reply:
FIXED